pkg/policy/validate.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/distribution/distribution/reference"
 	jsonpatch "github.com/evanphx/json-patch/v5"
+	"github.com/gardener/controller-manager-library/pkg/logger"
```

Comment:
Wrong logger lib got imported ?, we need to fix this.

Reply:
🙈

Reply:
https://github.com/kyverno/kyverno/pull/5311